BraintreeCore/src/main/java/com/braintreepayments/api/ApiClient.kt

```diff
@@ -0,0 +1,61 @@
+package com.braintreepayments.api
+
+import androidx.annotation.RestrictTo
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * @suppress
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class ApiClient constructor(private val braintreeClient: BraintreeClient) {
+
+    fun tokenizeGraphQL(tokenizePayload: JSONObject, callback: TokenizeCallback) {
+        braintreeClient.sendAnalyticsEvent("card.graphql.tokenization.started")
+        braintreeClient.sendGraphQLPOST(tokenizePayload.toString()) { responseBody, httpError ->
+            if (responseBody != null) {
+                try {
+                    callback.onResult(JSONObject(responseBody), null)
+                    braintreeClient.sendAnalyticsEvent("card.graphql.tokenization.success")
+                } catch (exception: JSONException) {
+                    braintreeClient.sendAnalyticsEvent("card.graphql.tokenization.failure")
+                    callback.onResult(null, exception)
+                }
+            } else {
+                braintreeClient.sendAnalyticsEvent("card.graphql.tokenization.failure")
+                callback.onResult(null, httpError)
+            }
```

Comment:
suggestion/question: for simple `if not null { ... } else { ... }` we could replace with `jsonBody?.also { ... } ?: { ... }`. Think its worth it? reference: https://stackoverflow.com/a/45878199

Reply:
i figured as well that when we start with coroutines, some major changes are going to be implemented here. Whatever you feel like!

Reply:
Would you be up for doing a PR into this branch with the change? This pattern will come up a lot once we start refactoring the clients. It'll be nice to lock in on a Kotlin strategy.